### PR TITLE
Travis-CI

### PR DESCRIPTION
**Pull request by kevgo**
Adds CI coverage to this library. Please note that there is a test that is flaky on CI, it should be fixed separately.

### CODE REVIEW

**Comment by charlierudolph** [comment]
Please disable email notifications

**Comment by kevgo** [comment]
All comments addressed

File: README.md [hunk 1]
```diff
@@ -1,9 +1,11 @@
-# go-execplus
+# go-execplus  [![Build Status](https://travis-ci.org/Originate/go-execplus.svg?branch=master)](https://travis-ci.org/Originate/go-execplus)
```
Comment (charlierudolph) [inline README.md]: Can we have the link go to `https://travis-ci.org/Originate/go-execplus/branches`? I don't like that clicking the link by default shows you latest build (which could be any branch). Travis doesn't appear to have a way to just see the build for a given branch. But the branches page should have the default branch listed at the top.

**Review by charlierudolph — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (kevgo) [reply README.md]: Done. That's an excellent idea, we should change all our repos to something like that. I personally find the "builds" tab more useful, and it also loads faster.

**Pull request closed without merging**